.github/workflows/csharp.yml

```diff
@@ -52,35 +56,40 @@ jobs:
       EVENT_TYPE: ${{ github.event_name }}
       CURRENT_BRANCH: ${{ github.head_ref || github.ref_name }}
     steps:
-      - uses: actions/checkout@v3
-      - name: Setup .NET
-        uses: actions/setup-dotnet@v1
-        with:
-          dotnet-version: 5.0.x
-      - name: Restore nuget packages
-        run: |
-          nuget restore OptimizelySDK.Travis.sln
-          nuget install ./OptimizelySDK.Tests/packages.config -OutputDirectory ./packages
-          nuget install NUnit.Runners -Version 2.6.4 -OutputDirectory ./testrunner
-      - name: script
-        run: |
-          ./install_mono.sh
-          xbuild /p:SignAssembly=true /p:AssemblyOriginatorKeyFile=$(pwd)/keypair.snk /p:Configuration=Release ./OptimizelySDK.Travis.sln
-          mono ./testrunner/NUnit.Runners.2.6.4/tools/nunit-console.exe ./OptimizelySDK.Tests/bin/Release/OptimizelySDK.Tests.dll
-      - name: find and sign dll
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1
+      - name: Setup NuGet
+        uses: NuGet/setup-nuget@v1
+      - name: Restore NuGet packages
+        run: nuget restore ./OptimizelySDK.Travis.sln
```

Comment:
Can you please run OptimizelySDK.sln instead of OptimizelySDK.Travis.sln

Reply:
The `OptimizelySDK.Travis.sln` does not include the demo app OptimizelySDK.DemoApp.csproj which has [build errors](https://github.com/optimizely/csharp-sdk/actions/runs/3470770177/jobs/5799387724#step:6:53).  

I'm guessing the demo app was not needed, so someone created the Travis version.  

In later sprints, I think we should fix or remove the demo app.

It's likely safe to continue to build the Travis solution instead.